packages/page-template-modal/src/styles/page-template-modal.scss

```diff
@@ -482,3 +448,8 @@ body.admin-bar:not( .is-fullscreen-mode ) .page-template-modal-screen-overlay {
 		flex-basis: auto;
 	}
 }
+.sidebar-modal-opener {
+	.template-selector-item__label {
+		max-width: 300px;
+	}
+}
```

Comment:
This rule might not belong here. You could maybe interpret it as an ETK specific override of the label style? I think maybe we should just leave it here though and tidy it up as part of all the CSS changes being made.